### PR TITLE
Automated build changes

### PR DESCRIPTION
**Pull request by ILMTitan**
Fix two tests where the gcloud config was not initialized. Fix a few warnings. Suppress one common warning type.

### CODE REVIEW

File: Google.PowerShell.IntegrationTests/Compute/Compute.GceRoute.Tests.ps1 [hunk 1]
```diff
@@ -123,9 +124,8 @@ Describe "Get-GceRoute" {
 
     It "should list for project" {
         $routes = Get-GceRoute
-
-        # the one we created, plus the default internet gateway route plus four subnetwork routes.
-        $routes.Count | Should Be 6
+        # the one we created, plus the default internet gateway route plus subnetwork (region) routes.
```
Comment (chrsmith) [inline Google.PowerShell.IntegrationTests/Compute/Compute.GceRoute.Tests.ps1]: Start sentences with a capital letter, "The".

File: Google.PowerShell.IntegrationTests/GcloudCmdlets.ps1 [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 # Install the GCP cmdlets module into the current PowerShell session.
 function Install-GcloudCmdlets() {
-    $dll = Get-ChildItem $PSScriptRoot\..\Google.PowerShell\bin -Recurse -Include Google.PowerShell.dll |
-        sort -Descending -Property LastWriteTime |
+    $dll = Get-ChildItem $PSScriptRoot\.. -Recurse -Include Google.PowerShell.dll |
```
Comment (chrsmith) [inline Google.PowerShell.IntegrationTests/GcloudCmdlets.ps1]: I assume this is a no-op change, and you are just prettying up the code?

File: Google.PowerShell/Google.PowerShell.csproj [hunk 3]
```diff
@@ -23,6 +23,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Debug\Google.PowerShell.XML</DocumentationFile>
+    <NoWarn>1591</NoWarn>
```
Comment (chrsmith) [inline Google.PowerShell/Google.PowerShell.csproj]: Please add an XML doc comment about what the warning is. As looking at the file you don't know if 1591 is important or not.

**Comment by chrsmith** [comment]
Some minor issues. Thanks for the cleanup CL.

File: Google.PowerShell.IntegrationTests/GcloudCmdlets.ps1 (hunk 2 above)
Comment (ILMTitan) [inline Google.PowerShell.IntegrationTests/GcloudCmdlets.ps1]: Yes.

**Comment by ILMTitan** [comment]
PTAL

File: Google.PowerShell/Google.PowerShell.csproj [hunk 4]
```diff
@@ -23,6 +23,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Debug\Google.PowerShell.XML</DocumentationFile>
+    <!--Don't create warnings for uncommented public members.-->
```
Comment (chrsmith) [inline Google.PowerShell/Google.PowerShell.csproj]: Nit: I'd put space after `<--` and before `-->` for readability. Also, including the exact warning text is useful as well. But ultimately it doesn't matter.

**Comment by chrsmith** [comment]
:ship:

**Pull request merged**